cmd/planner-agent/main.go

```diff
@@ -77,9 +76,9 @@ func (a *agentCmd) Execute() error {
 	undo := zap.ReplaceGlobals(logger)
 	defer undo()
 
-	agentID, err := a.readFile(agentFilename)
+	agentID, err := a.readPersistentFile(agentFilename)
 	if err != nil {
-		zap.S().Fatalf("failed to retreive agent_id: %v", err)
+		zap.S().Fatalf("failed to retrieve agent_id: %v", err)
```

Comment:
in another PR

Reply:
:/ I thought I will sneak this one line change - but ok.